Smartsheet.Net.Standard/Entities/Share.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using Smartsheet.Net.Standard.Interfaces;
+
+namespace Smartsheet.Net.Standard.Entities
+{
+    public class Share : ISmartsheetObject
+    {
+        public string Id { get; set; }
+        public long? GroupId { get; set; }
+        public long? UserId { get; set; }
+        public string Type { get; set; }
+        public string AccessLevel { get; set; }
+        public bool? CCMe { get; set; }
```

Comment:
I think this needs to be `CcMe` to serialize/deserialize correctly.

Reply:
Updated